### PR TITLE
Quadrat: Update the layout of the query template

### LINKED ISSUE #3804: Quadrat: Post list cleanup
I have a few suggestions for more closely aligning the list of posts in Quadrat with the full post view: 

- Move the date up to the top, center-align the text.
- Move the title below that, center-align the text.
- The spacing below the post title is currently 30px. Let's increase that to 45px. 
- Increase the spacer block underneath each post from 30px to 60px to allow for more whitespace in between each one. 

Before|After
---|---
Screen Shot 2021-05-10 at 11 59 22 AM|Screen Shot 2021-05-10 at 12 15 41 PM

cc @beafialho

### PR DESCRIPTION
**Pull request by scruffian**
#### Changes proposed in this Pull Request:
This makes that changes suggested in #3804 

- Move the date up to the top, center-align the text.
- Move the title below that, center-align the text.
- Increase the spacer block underneath each post from 30px to 60px to allow for more whitespace in between each one.
- The spacing below the post title is currently 30px. Let's increase that to 45px.

This last point was achieved by adding a 15px spacer rather than updating the margin on the block.

Screenshot 2021-05-26 at 16 31 16

#### Related issue(s):
#3804

### CODE REVIEW

**Comment by jffng** [comment]
When there is no featured image, the excerpt feels far from the post title, like it's floating:

Screen Shot 2021-05-26 at 12 51 27 PM

cc @kjellr

**Comment by kjellr** [comment]
> When there is no featured image, the excerpt feels far from the post title, like it's floating:

Yeah, can we add that 15px of top spacing to the featured image block itself? Or hide it when the featured image is not present?

---

Also, the post title block appears and takes up a lot of space even when there's no post title: 

Screen Shot 2021-05-27 at 8 20 07 AM
Screen Shot 2021-05-27 at 8 20 12 AM

I'm not sure if that's something we should fix in the theme, or in Gutenberg though.

File: quadrat/sass/theme.scss [hunk 1]
```diff
@@ -19,10 +19,11 @@
...
 @import "elements/forms";
-@import "header";
-@import "meta";
-@import "post-header";
-@import "footer";
+@import "templates/header";
+@import "templates/footer";
+@import "templates/meta";
+@import "templates/post-header";
+@import "templates/query";
```
Comment (MaggieCabrera) [inline quadrat/sass/theme.scss:26]: thank you for this

**Review by MaggieCabrera — APPROVED** [review]
LGTM!

**Pull request merged**